langmeta/shared/src/main/scala/org/langmeta/semanticdb/ResolvedSymbol.scala

```diff
@@ -1,9 +1,65 @@
 package org.langmeta
 package semanticdb
 
-final case class ResolvedSymbol(symbol: Symbol, denotation: Denotation) {
+import scala.runtime.AbstractFunction2
+
+final case class ResolvedSymbol2(symbol: Symbol, denotation: Denotation, members: List[Signature]) {
```

Comment:
Is this class used anywhere?

Reply:
Reverted ResolvedSymbol to original design.